### DISCUSSION TITLE
Modified files after npm ci

### DISCUSSION THREAD

**Discussion by astridx** [joomla/joomla-cms#32760] [Q&A] [answered]
After I recloned the 4.0-dev branch and then ran `npm ci`, changes appear when I do call `git status`. 

For example, in file `/plugins/editors/codemirror/codemirror.xml,` the line
`<version>5.59.4</version>`
changed to
`<version>5.60.0</version>`.
 
Am I doing something wrong? Actually, running` npm ci `should not cause any changes, right?

**Comment by dgrammatiko** [comment, marked as answer]
> Am I doing something wrong? Actually, runningnpm cishould not cause any changes, right?

No, you're not doing something wrong, but whoever updated the package.json last time forgot to run `npm install` and commit the changes. At some point, this should become a github action (it can't right now because the 4.0-dev is not the default branch)

**Comment by astridx** [comment]
@dgrammatiko Thank you
